Servers/UI/OJS.Servers.Ui/ClientApp/src/components/contests/contest/Contest.tsx

```diff
@@ -108,6 +128,9 @@ const Contest = () => {
                     {contest?.name}
                 </Heading>
                 <Heading type={HeadingType.secondary} className={secondaryHeadingClassName}>
+                    {isOfficial
+                        ? renderParticipants(activeParticipants, true)
+                        : renderParticipants(totalParticipants, false)}
```

Comment:
We don't do this by convention, i.e. login inside the HTML.

Also, it is not necessary and this the following is enough:
```
{renderParticipants(activeParticipants, isOfficial)}
```